src/app/containers/CpsAssetPageMain/index.jsx

```diff
@@ -22,6 +23,9 @@ const CpsAssetPageMain = ({ pageData }) => {
       <LinkedData type="Article" seoTitle={title} />
       <Grid as="main" role="main">
         <GridItemConstrainedMedium>
+          <Link to="/pidgin/test-12345678" data-e2e="cpsAssetDummyLink">
```

Comment:
Prior to merge, can you replace `/pidgin/test-12345678` with a real asset please :)